relayer/.env.example

```diff
@@ -0,0 +1,6 @@
+PARACHAIN_RELAY_ASSETHUB_ETH_KEY=0x3646505e08a0f3a61417d91db18f2911b2d17c01563044f3e2f106bf36679a6a
+EXECUTION_RELAY_ASSETHUB_SUB_KEY=//ExecutionRelayAssetHub
```

Comment:
rather leave these unset in case someone mistakenly puts real secrets here

Reply:
https://github.com/Snowfork/snowbridge/pull/1269/commits/756c0bc23e80896c64e0e8da9d85c9acbf998b76